lib/diagnostic.js

```diff
@@ -28,7 +37,12 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
+let owner = {
+   name: Mario;
```

Comment:
This should be ```name: 'Mario',```.  The way that it's currently written will cause this to throw an error because it will look for something that doesn't exist called Mario and also because `;` is not a valid delimiter for object properties.